spec/lob/v1/bank_account_spec.rb

```diff
@@ -111,4 +111,22 @@
       end
     end
   end
+
+  describe "verify" do
+    it "should verify a bank account" do
+      VCR.use_cassette('verify_bank_account') do
+        new_bank_account = subject.bank_accounts.create(
+          routing_number: @sample_bank_account_params[:routing_number],
+          bank_address: @sample_address_params.clone,
+          account_number: @sample_bank_account_params[:account_number],
+          account_address: @sample_address_params.clone,
+          signatory: "John Doe"
+        )
+
+        delete_result = subject.bank_accounts.verify(new_bank_account["id"], amounts: [0.12, 0.34])
```

Comment:
Should the wrapper accept the amounts as floats or ints? cause the api accepts ints, right?